prime-angular-frontend/src/app/shared/components/forms/site-information-form/site-information-form.component.ts

```diff
@@ -86,41 +86,39 @@ export class SiteInformationFormComponent implements OnInit {
     if (change.checked) {
       this.activeBeforeRegistration.setValue(false);
       this.isNewWithoutSiteId.setValue(false);
-      this.setCommunityPharmacySiteIdPrefix();
     }
-    this.updatePECValidator();
+    this.updatePEC();
   }
 
   public checkAsIsNewWithoutSiteId(change: MatCheckboxChange): void {
     if (change.checked) {
       this.activeBeforeRegistration.setValue(false);
       this.isNewWithSiteId.setValue(false);
-      this.pec.setValue("");
     }
-    this.updatePECValidator();
+    this.updatePEC();
   }
 
   public checkAsOperational(change: MatCheckboxChange): void {
     if (change.checked) {
       this.isNewWithoutSiteId.setValue(false);
       this.isNewWithSiteId.setValue(false);
-      this.setCommunityPharmacySiteIdPrefix();
     }
-    this.updatePECValidator();
+    this.updatePEC();
   }
 
-  private setCommunityPharmacySiteIdPrefix() {
-    if (!this.pec.value || this.pec.value === "") {
-      this.pec.setValue("BC00000");
-    }
-  }
 
-  private updatePECValidator(): void {
+  private updatePEC(): void {
     if (this.careSettingCode === CareSettingEnum.COMMUNITY_PHARMACIST) {
       if (this.activeBeforeRegistration.value || this.isNewWithSiteId.value) {
         this.formUtilsService.setValidators(this.pec, [Validators.required, FormControlValidators.communityPharmacySiteId]);
+        this.pec.enable();
+        if (!this.pec.value || this.pec.value === "") {
+          this.pec.setValue("BC00000");
+        }
       } else {
         this.formUtilsService.setValidators(this.pec, [FormControlValidators.communityPharmacySiteId]);
```

Comment:
Wondering, do we still need a validator in this case?

Reply:
the validation is removed.